extension/encoding/googlecloudlogentryencodingextension/log_proto_payload_test.go

```diff
@@ -184,62 +189,6 @@ func TestProtoFieldTypes(t *testing.T) {
 	}
 }
 
-func TestProtoErrors(t *testing.T) {
```

Comment:
Proto errors were no longer valid (they affected cloud audit logs), so test can be safely removed